Test/JsonDiffTest.cs

```diff
@@ -0,0 +1,34 @@
+using c_sharp_json_diff;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Test
+{
+    public class JsonDiffTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void WhenJsonDiffOnTwoStringsThenCorrectValueReturned()
+        {
+            string left = @"{
```

Comment:
This test is a good start, but the next level is to refactor it as follows:

1. Parameterise the test on 3 different parameters: the left JSON, the right JSON and the expected result JSON- all strings
2. The actual test code itself then would perform the numeric diff on left and right, the same as it does now
3. Finally, the assertion would be simple: just serialize the result of the numeric diff, and check that the string is equal to the expected string. This is much more powerful and general purpose than the existing assertions, because it includes all of the existing assertions as special-cases. The only thing you'd need to watch out for is formatting: maybe you can remove all whitespace from the expected & actual strings before comparing to get around this.

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix

Reply:
Yeah I'll change that. This was a quick test to see if the code was working as expected or not